packages/ui/src/components/Visualization/Custom/hooks/paste-step.hook.tsx

```diff
@@ -3,54 +3,75 @@ import { AddStepMode, IVisualizationNode } from '../../../../models/visualizatio
 import { EntitiesContext } from '../../../../providers/entities.provider';
 import { ClipboardManager } from '../../../../utils/ClipboardManager';
 import { CatalogModalContext } from '../../../../providers/catalog-modal.provider';
+import { IClipboardCopyObject } from './copy-step.hook';
 
 export const usePasteStep = (vizNode: IVisualizationNode, mode: AddStepMode) => {
   const entitiesContext = useContext(EntitiesContext)!;
   const catalogModalContext = useContext(CatalogModalContext);
   const [isCompatible, setIsCompatible] = useState(false);
+  const permissionName = 'clipboard-read' as PermissionName;
```

Comment:
[💅] Please let's move this line outside of the function definition, so we don't create this string on every execution 🙏